flepimop/gempyor_pkg/src/gempyor/shared_cli.py

```diff
@@ -0,0 +1,295 @@
+"""
+An internal module to share common CLI elements. Defines the overall cli group,
+supported options for config file overrides, and custom click decorators.
+"""
+
+import multiprocessing
+import pathlib
+import warnings
+from typing import Callable, Any
+import re
+
+import click
+import confuse
+
+from .utils import config, as_list
+
+__all__ = []
+
+
+@click.group()
+@click.pass_context
+def cli(ctx: click.Context) -> None:
+    """Flexible Epidemic Modeling Platform (FlepiMoP) Command Line Interface"""
+    pass
+
+
+class AlphanumericParamType(click.ParamType):
```

Comment:
Do we still need this type with the new way of calling the thing ? click should ensure strings no ?

Reply:
I have made this change; I continue to assess that filtering to subset of allowed characters here would be preferrable, but we can institute that change when something actually blows up over it.